plejd/ble.bluez.js

```diff
@@ -460,20 +443,37 @@ class PlejdService extends EventEmitter {
     this.emit('pingSuccess', pong[0]);
   }
 
-  async startWriteQueue() {
+  startWriteQueue() {
     console.log('startWriteQueue()');
-    clearInterval(this.writeQueueRef);
+    clearTimeout(this.writeQueueRef);
 
     this.writeQueueRef = setTimeout(() => this.runWriteQueue(), this.writeQueueWaitTime);
   }
 
   async runWriteQueue() {
```

Comment:
runWriteQueue is now in charge of finding errors and evaluate what to do with them.

So first we see if we have any retries to run, and do it. (only way to handle retries without even more of an overhaul).
If they error again, lets throw the message away.

Then we run the normal queue, if any writes throw an error we put them in the retry-queue.

If we have anything in the retry-q, we do not queue another writeQueue-timeout and instead execute an timeouted init (no more need of the throttled version).